apps/example-app/src/app/issues/issue-106.spec.ts

```diff
@@ -1,46 +0,0 @@
-import { Component } from '@angular/core';
```

Comment:
Why are these issue reproductions removed?

Reply:
After giving this a second look, these tests don't really add any value anymore.
They're used as reproductions for issues, most of the time they are aimed to a very specific use case. It would've been more useful to name these tests instead of using `issue-#` 😅.

So 👍 to remove them.